### PR TITLE
fix(rmo): make more boilerplatey

### PR DESCRIPTION
**Pull request by georgettica**
this will make the CI more conformant to boilerplate

### CODE REVIEW

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml [hunk 1]
```diff
@@ -3,19 +3,12 @@ base_images:
     name: ubi
     namespace: ocp
     tag: "8"
-  openshift_release_golang-1.13:
```
Comment (2uasimojo) [inline ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:6]: Just make everything before `resources` look exactly like https://github.com/openshift/release/blob/master/ci-operator/config/openshift/configure-alertmanager-operator/openshift-configure-alertmanager-operator-master.yaml

**Review by 2uasimojo — CHANGES_REQUESTED** [review]

**Review by 2uasimojo — APPROVED** [review]
I think this should do it. Make sure rehearsals pass, but

/lgtm

**Comment by georgettica** [comment]
/hold
so it doesn't pass 100% now, but I can debug rehearsals

**Comment by georgettica** [comment]
fingers crossed
/unhold

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml [hunk 2]
```diff
@@ -1,35 +1,15 @@
...
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.14
+  from_repository: true
 images:
 - dockerfile_path: Dockerfile
-  from: base
-  to: route-monitor-operator
+  to: unused
```
Comment (petr-muller) [inline ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:5]: why keep it then?

Comment (2uasimojo) [reply ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:5]: Are you asking why we keep the image build when we’re not using the result? It’s to make sure it succeeds. I’m not sure about rmo (it’s a snowflake in several ways) but the other osd operators don’t have any other mechanism to validate the docker build at this stage.

**Comment by georgettica** [comment]
seems the installation of the aws cluster failed.. will wait for it to fully crash so I can recreate

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml (hunk 2 above)
Comment (georgettica) [reply ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:5]: @petr-muller is we are deleting this, how would I test this works?

Comment (petr-muller) [reply ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:5]: yep makes sense

**Comment by georgettica** [comment]
Seems there is no quota to provision the cluster..
Can someone push this Friday?

**Comment by georgettica** [comment]
Let's retest now just in case
/retrst

**Review by 2uasimojo — COMMENTED** [review]
This looks ready. 

/lgtm

**Pull request merged**